.gitignore

```diff
@@ -0,0 +1,7 @@
+*.txt
+*.env
```

Comment:
Are you sure  you want to ignore such important files. All these types of files are important for any project.